ocs_ci/deployment/deployment.py

```diff
@@ -216,6 +217,18 @@ def deploy_ocs(self):
                 f"MDS deployment Failed! Please check logs!"
             )
 
+        # WA for bug: https://bugzilla.redhat.com/show_bug.cgi?id=1747388
+        cluster = CephCluster()
+        wa_cmd = (
+            "for POOL in ocsci-cephfs-metadata ocsci-cephfs-data0; do for PGN"
```

Comment:
Not sure about the static *pool* names, I think its more accurate to set the default pg as well as pgp_num using config overrides.

Reply:
@vasukulkarni , please see my answer in the mail thread. Do not want to put a lot of effort here and this is almost verified with normal deployment and also via operator.

Reply:
@petr-balogh I can send out an PR for ConfigOverrides as I feel its accurate and it handles all the pools(dynamically created in tests)? from you own console logs, we are creating extra rbd pool for which default number is 8/8,  In this PR we are just handling for one pool, I hope you understand its not just about deployment test.

```
14:00:51 - MainThread - tests.helpers - INFO - POOLS are [{'poolnum': 1, 'poolname': 'ocsci-cephfs-metadata'}, {'poolnum': 2, 'poolname': 'ocsci-cephfs-data0'}, {'poolnum': 7, 'poolname': 'cbp-test-2684073c19e147b888b6eb82b967f029'}]
```

Reply:
I see that ConfigOverrides is not honored yet - https://github.com/rook/rook/issues/3752

Reply:
Just checked also the WA discussed with apply. When I tried to  do some changes via ConfigOverrides and used oc apply command to do those changes it didn't actually apply to those pools created during the deployment.  Not sure about the mentioned issue and the PR  https://github.com/rook/rook/pull/3769 should fix this behavior. But if it won't be fixed soon the provided WA is still valid and we need to add one more wit apply and ConfigOverrides like:
```yaml
spec:
  configOverrides:
    - who: global
      option: osd pool default pg num
      value: "100"
    - who: global
      option: osd pool default pgp num
      value: "100"
```
tom make those default also for the new pools created by tests.

I see we had just 5 failures but don't look as related to this pg num issue,  when I tool a look a a glance here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/720/testReport/.

@vasukulkarni up to you to merge this one now and do the apply fix on top of this one.

Reply:
It looks like it may take 2 more days, based on results merging now and can experiment with autoscale as top up.